pod-resizer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Copyright 2016 The Kubernetes Authors. All rights reserved.
```

Comment:
Please stick the full copyright text here (like in https://github.com/kubernetes/contrib/blob/master/ingress/images/nginx-slim/Dockerfile).